### DISCUSSION TITLE
How to use PDNSIM in OPenRoad-flow-scripts

### DISCUSSION THREAD

**Discussion by Coin-Hou** [The-OpenROAD-Project/OpenROAD-flow-scripts#346] [Q&A] [answered]
The Makefile doesn't include the code for PDNSIM, please help.

**Comment by maliberty** [comment]
You can see the docs at https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/src/psm

**Reply by Coin-Hou** [reply]
Thanks for the reply. For my first time using OpenROAD, I still have trouble executing PDNSim on asap7 tech. The code stopped before outputting the results with lots of warnings about [unconnected instance]. I think maybe there's something wrong with reading the .lef/.lib profile. Anyway, my codes work for Nangate45/SKY130 but not for asap7, although they are similar.

**Comment by maliberty** [comment, marked as answer]
There was an issue with asap7 pdn configuration fixed recently (https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/commit/dda86f631047ee1a7215504c8ec8b4478034fc35) .  Are you using the latest ORFS?

**Reply by maliberty** [reply]
If you are a test case would be helpful.

**Reply by Coin-Hou** [reply]
Thanks for your help. It works.
